### ISSUE TITLE
FloorspaceJS with a map doesn't work

### ISSUE THREAD

**Issue by jmarrec** [openstudiocoalition/OpenStudioApplication#364]
Issue overview
--------------

```
DevTools listening on ws://127.0.0.1:41075/devtools/browser/a9855131-5707-45ec-aca8-e6e95e592bb5
Critical: Access to image at 'https://c.tile.openstreetmap.org/18/54623/99464.png' from origin 'qrc:' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
Critical: Access to image at 'https://c.tile.openstreetmap.org/18/54624/99465.png' from origin 'qrc:' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
```

![image](https://user-images.githubusercontent.com/5479063/120218978-cdb4c500-c23a-11eb-83e7-74085b489675.png)

## Steps to Reproduce

1.
2.
3.
4.

### Environment

Some additional details about your environment for this issue (if relevant):
 - Platform (Operating system, version): Ubuntu 18.04
 - Version of OpenStudioApplication (if using an intermediate build, include SHA): v1.2.0-rc4

**Comment by macumber** [comment]
Duplicate of #366
